src/MovieDetails.js

```diff
@@ -7,7 +7,7 @@ const MovieDetails = (props) => {
       <section>
         <button className='back-to-main-button' onClick={() => props.displayMain()}>Back to Main</button>
         <img className='movie-backdrop' src={props.selectedMovie.backdrop_path} alt={props.selectedMovie.title} />
-        <p className='gradient'></p>
+        // <p className='gradient'></p>
```

Comment:
We love the gradient but realize this p tag is actually causing functionality issues when the screen size chages.